client/core/core_test.go

```diff
@@ -9168,12 +9168,12 @@ out:
 			t.Fatalf("timed out waiting for synced wallet note. Received %d progress notes", progressNotes)
 		}
 	}
-	// Should get 9 notes, but just make sure we get at least half of them to
-	// avoid github vm false positives.
-	if progressNotes < 5 /* 9? */ {
-		t.Fatalf("expected 9 progress notes, got %d", progressNotes)
+	// We must get at least 1 progress note.
+	// By the time we've got 9th note it should signal that the wallet has been
+	// synced (due to how we've set up testDuration and syncTickerPeriod values).
+	if progressNotes < 1 || progressNotes > 9 {
```

Comment:
If we got here at all, that mean's we got at least one note (otherwise we'd hit the `t.Fatal` above on timeout), and the last note had `walletNote.Wallet.Synced = true` (otherwise no `break out`).

Since this test has degenerated completely now, we might as well remove this note count check entirely.